modules/ml/src/test/java/org/apache/ignite/ml/naivebayes/compound/Data.java

```diff
@@ -0,0 +1,69 @@
+package org.apache.ignite.ml.naivebayes.compound;
```

Comment:
Could you please add Apache header containing the license?

Reply:
ditto.